### PR TITLE
LRAC-8661,LRAC-8662,LRAC-8663,LRAC-8664,LRAC-8665,LRAC-8666,LRAC-8668,LRAC-8669 | master

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LRAC-8661
https://issues.liferay.com/browse/LRAC-8662
https://issues.liferay.com/browse/LRAC-8663
https://issues.liferay.com/browse/LRAC-8664
https://issues.liferay.com/browse/LRAC-8665
https://issues.liferay.com/browse/LRAC-8666
https://issues.liferay.com/browse/LRAC-8668
https://issues.liferay.com/browse/LRAC-8669

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase [hunk 1]
```diff
@@ -30,21 +36,157 @@ definition {
...
+
+			ACNavigation.goToSettings();
+
+			Pause(locator1 = "10000");
+
+			var expectedChannelId = ACSettings.getPropertyId(propertyName = "Liferay DXP (205) Combined Property");
+		}
+
+		task ("Get the DXP Instance ID in AC") {
+			var dataSourceName = StringUtil.extractFirst("Liferay DXP (205) Combined Property", "Combined");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase:135]: Hi @Hanlf , I think you forgot the data source name is hard coded. Could you please fix this line from:

`var dataSourceName = StringUtil.extractFirst("Liferay DXP (205) Combined Property", "Combined");`

to

`var dataSourceName = StringUtil.extractFirst("${propertyName}", "Combined");`

Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase [hunk 2]
```diff
@@ -57,57 +199,451 @@ definition {
...
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase:596]: Hi @Hanlf , I think you forgot the data source name is hard coded. Could you please fix this line from:

`var dataSourceName = StringUtil.extractFirst("Liferay DXP (205) Combined Property", "Combined");`

to

`var dataSourceName = StringUtil.extractFirst("${propertyName}", "Combined");`

Thanks!

**Comment by alexcandidos** [comment]
hi @Hanlf , could you please do the adjustment I commented on? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase [hunk 3]
```diff
@@ -57,57 +199,451 @@ definition {
...
+		}
+
+		task ("Get the Property ID in AC") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			Pause(locator1 = "10000");
+
+			var expectedChannelId = ACSettings.getPropertyId(propertyName = "Liferay DXP (205) Combined Property");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase:592]: The same here, I think it should be:

`var expectedChannelId = ACSettings.getPropertyId(propertyName = "${propertyName}");`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase [hunk 4]
```diff
@@ -30,21 +36,157 @@ definition {
...
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase:132]: The same here, I think it should be:

`var expectedChannelId = ACSettings.getPropertyId(propertyName = "${propertyName}");`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase (hunk 1 above)
Comment (Hanlf) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase]: Ah, sorry, I forgot to update it, thanks for catching it!

**Comment by Hanlf** [comment]
@alexcandidos  updated, please review!

**Comment by alexcandidos** [comment]
Hi @Hanlf , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**